doc/integrating_applications/topics/transforming_target_data.adoc

```diff
@@ -0,0 +1,16 @@
+[id=transforming-target-data]
+== Transforming target data
+
+Adding a transformation to the target data defines how the data will be stored at the target.
```

Comment:
Maybe in place of "how the data will be stored at the target" we should say:
"the format of the data for the next connection or step."  ???